### DISCUSSION TITLE
Does SagerNet need a rooted device?

### DISCUSSION THREAD

**Discussion by wenjucktes01** [SagerNet/SagerNet#490] [Q&A] [answered]
I looked for many porgams to let the proxy get to the wohle device, the problem  most apps need an rooted Phone. So my question is dose SagerNet need a rooted phone

**Comment by iKirby** [comment, marked as answer]
It does not need a rooted device. When starting service for the first time, you'll see a connection request dialog, just click OK to allow the app to create VPN connection.
